{{ cookiecutter.project_shortname }}/.travis.yml

```diff
@@ -24,12 +24,12 @@ python:
 before_install:
   - "travis_retry pip install --upgrade pip setuptools py"
   - "travis_retry pip install twine wheel coveralls requirements-builder"
-  - "requirements-builder --level=min setup.py > .travis-lowest-requirements.txt"
-  - "requirements-builder --level=pypi setup.py > .travis-release-requirements.txt"
-  - "requirements-builder --level=dev --req requirements-devel.txt setup.py > .travis-devel-requirements.txt"
+  - "requirements-builder --level=min setup.py > built-requirements-lowest.txt"
```

Comment:
BTW @jirikuncar FYI I mentioned to @krzysztof IRL that we could perhaps reconsider the `--level` values in `requirements-builder` too, if we are entering this renaming game.  The reason being is that while `min` and `dev` are perfectly self-understandable, `pypi` may not be, because `min` is *also* on PyPI, just like `pypi` is.  Hence something like your `release`, or a new `max`, may be a more appropriate.  (`min/max/dev`)

If we do change the names, then we can consider changing them also on `requirements-builder`...